### ISSUE TITLE
Python 3: comtypes.client.PumpEvents raises error after timeout

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9921]
### Steps to reproduce:
Open Outlook.
When there is a message in the open folder
Nvda shows a window called waiting for outlook.
It happens that in previous verifications of the nvda this window appears more exits automatically.
In the verdict of nvda that I am testing this window opens more does not date.
It only closes if I press alt + f4.
### Actual behavior:
the window waiting for the Outlook is displayed but does not close automatically
### Expected behavior:
the window waiting for the outlook is opened more automatically closes.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
threshold_py3_staging-18054,92458e98
#### Windows version:
10 18362.175
#### Name and version of other software in use when reproducing the issue:
outlook Office 365 MSO (16.0.11629.20238) 64 bits 
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
alpha-17697,1d50498c
works correctly.
log:
[outlook.txt](https://github.com/nvaccess/nvda/files/3388011/outlook.txt)

**Comment by josephsl** [comment]
Hi,

For those not speaking Portuguese, the traceback ends with:
OSError: [WinError -2147417835] OLE has sent a request and is waiting for a reply

Thanks.

**Comment by leonardder** [comment]
I've seen this once now, but haven't found a reliable way to reproduce this.

**Comment by leonardder** [comment]
It turns out that comtypes.client.PumpEvents raises an error after timing out. This can easily be reproduced from the python console.

**Comment by leonardder** [comment]
The issue is in comtypes.client.events around line 278

```
        except WindowsError as details:
            if details.args[0] != RPC_S_CALLPENDING: # timeout expired
                raise
```

**Comment by feerrenrut** [comment]
Fixed with commit d55bee3 via PR #9950
